internal/cli/atlas/search/index_opts.go

```diff
@@ -24,59 +24,59 @@ import (
 	atlasv2 "go.mongodb.org/atlas-sdk/v20230201006/admin"
 )
 
-const defaultAnalyzer = "lucene.standard"
+const DefaultAnalyzer = "lucene.standard"
 const deprecatedFlagMessage = "please use --file instead"
 
 type IndexOpts struct {
-	name           string
-	dbName         string
-	collection     string
-	analyzer       string
-	searchAnalyzer string
-	dynamic        bool
-	fields         []string
-	filename       string
-	fs             afero.Fs
+	Name           string
+	DBName         string
+	Collection     string
+	Analyzer       string
+	SearchAnalyzer string
+	Dynamic        bool
+	Fields         []string
```

Comment:
these are kept for backward compatibility but we shouldn't use them in new comamnds

Reply:
thanks for catching this, made the field private. The deployments command was not setting index fields, but it's better to keep it private to avoid accidentally setting it